ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -198,7 +198,18 @@ powervs*)
     export IBMCLOUD_API_KEY
     ;;
 nutanix) export TEST_PROVIDER='{"type":"nutanix"}' ;;
-external) export TEST_PROVIDER='' ;;
+external)
+    # FIXME(mtulio): https://issues.redhat.com/browse/OCPBUGS-53249
+    # Forcing openshift/origin presubmits to set the flag provider to "external" to validate the PR
+    # https://github.com/openshift/origin/pull/29623
+    # Presubmits on origin repo is currently permanent failing, skips is addressed on OCPBUGS-53249, required by
+    # https://github.com/openshift/kubernetes/pull/2247
+    if [[ $JOB_NAME == *"pull-ci-openshift-origin-release"* ]]; then
```

Comment:
maybe `pull-ci-openshift-origin`?

Reply:
based in https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/29623/pull-ci-openshift-origin-main-e2e-external-aws-ccm/1904250996830244864